### PR TITLE
Done

### PR DESCRIPTION
**Pull request by jorshimayor**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

[x] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
[x] The title of the Pull Request matches the title of my tutorial.
[x] I've followed the tutorial template structure.
[x] I've proofread the tutorial for spelling and grammar mistakes.
[x] I've thoroughly explained the code, with inline code comments and text explanations.
[x] I've run a linter & prettifier to format my code blocks.
[x] I've run the tutorial to test it and the code works.
[x] I've followed all the best practices mentioned in the [here](https://docs.celo.org/community/celo-sage/complete-your-tutorial#best-practices).

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/05-05-2023-creating-a-custom-token-on-celo/2023-04-30-creating-a-custom-token-on-celo.md
Comment (Julius170) [inline blog/05-05-2023-creating-a-custom-token-on-celo/2023-04-30-creating-a-custom-token-on-celo.md:1]: Hi @jorshimayor, Your file upload format is wrong, your .md file shouldn't have the date format at the start of the file name

"blog/05-05-2023-creating-a-custom-token-on-celo/creating-a-custom-token-on-celo"

File: blog/05-05-2023-creating-a-custom-token-on-celo/creating-a-custom-token-on-celo.png.png
Comment (Julius170) [inline blog/05-05-2023-creating-a-custom-token-on-celo/creating-a-custom-token-on-celo.png.png:1]: this Image should be inside the 'data-tutorials/showcase/intermediate/" folder

And also you have ".png.png" ending the file, you should reduce it to just ".png"

File: docs-2
Comment (Julius170) [inline docs-2:1]: I'm not sure what these changes are but you're going to have to remove them

File: docs-1
Comment (Julius170) [inline docs-1:1]: I'm not surewhat these changes are, but you're going to have to remove them

File: src/data-tutorials/users.tsx
Comment (Julius170) [inline src/data-tutorials/users.tsx:1]: there are multiple changes here that shouldn't be there, you should have only the metadata for your tutorial on this file

**Review by Julius170 — COMMENTED** [review]
Hi @jorshimayor there's quite a few changes you're going to have to make on you PR, 

I've left comments to help you out with it...
can't wait to hear form you

**Review by Julius170 — COMMENTED** [review]
Hi @jorshimayor there's quite a few changes you're going to have to make on you PR, 

I've left comments to help you out with it...
can't wait to hear form you

**Comment by viral-sangani** [comment]
Closing Celo Sage PRs

**Pull request closed without merging**